src/views/Picklists.vue

```diff
@@ -51,9 +51,11 @@ export default defineComponent({
     })
   },
   updated () {
-    this.store.dispatch('picklist/findPickList').catch(err =>
-      this.store.dispatch('picklist/clearPicklist')
-    )
+    if(this.$route.path != "/tabs/settings") {
```

Comment:
This condition will not handle the case when we will have a new route.

Reply:
Sure, sir, will look into it.